### DISCUSSION TITLE
Unable to push to my repository

### DISCUSSION THREAD

**Discussion by benedwards195** [community/community#76883] [Programming Help]
### Help

Question

### Body

I am constantly finding new issues when pushing to a repository of mine, with the latest one being as screenshotted; the terminal server just stops there. Please can someone help me out? 
<img width="494" alt="Screenshot 2023-11-24 at 15 07 47" src="https://github.com/community/community/assets/9057647/3f19ce88-c0fe-4838-89b2-6699077892f7">

**Comment by deep100x** [comment]
Use Github Desktop, that better and more easy way to make repositories, commiting and push.

**Comment by hakimlutfi46** [comment]
that usually happens to me often too. If I experience that I usually do this

go to the local repo file -> then delete the .git folder -> then do the same thing as if you wanted to push to the repository. If the same thing still happens. Try using githubdeksop, it will help you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @benedwards195 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
You can also remove your origin by writing this command:
`git remote remove origin`. 
Also you can use GitHub Desktop to efficiently to solve that in a GUI interface.
